.github/workflows/deploy-to-wordpress-dotorg.yml

```diff
@@ -0,0 +1,16 @@
+name: Deploy to WordPress.org
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  tag:
+    name: New tag
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: WordPress Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@master
+      env:
+        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
```

Comment:
Setting up same for peterwilsoncc/rapid-canoncial-urls I needed to include `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` for the commit to work in each of these config files. I'm not sure why but I think it may have something to do with re-checkout. 🤷🏻‍♂️